dotnet/src/Functions/Functions.OpenApi.Extensions/Extensions/ApiManifestKernelExtensions.cs

```diff
@@ -109,11 +109,16 @@ await DocumentLoader.LoadDocumentFromUriAsStreamAsync(new Uri(apiDescriptionUrl)
                 DocumentLoader.LoadDocumentFromFilePathAsStream(parsedDescriptionUrl.LocalPath,
                     logger);
 
-            var documentReadResult = await new OpenApiStreamReader(new()
-            {
-                BaseUrl = new(apiDescriptionUrl)
-            }
-            ).ReadAsync(openApiDocumentStream, cancellationToken).ConfigureAwait(false);
+            // TODO: Refactor the code to the new readers available in the OpenAPI.NET V2
+            ReadResult documentReadResult = await OpenApiModelFactory.LoadAsync(
+                input: openApiDocumentStream,
+                format: "TBD",
```

Comment:
Is this parameter required? can be ommited with a comment?

```suggestion
                format: "TBD",
```

Reply:
Unfortunately, it's required and can't be omitted. It'll be handled in scope of this task - https://github.com/microsoft/semantic-kernel/issues/9859